index.js

```diff
@@ -13,9 +14,16 @@ app.use(bodyParser.json())
 app.use(bodyParser.urlencoded({extended: true}))
 
 const routes = require('./routes')
-
+app.use(errorHandlers.productionErrors)
```

Comment:
when i said: change error handling, i mean all the error-related stuff, all the code that is already after app.listen

Reply:
solved